charts/render.py

```diff
@@ -27,17 +28,22 @@
 from yaml.representer import SafeRepresenter
 
 
-BOILERPLATE = '''
+BOILERPLATE = """
```

Comment:
Even though I personally prefer `"""` over `'''`, let's be consistent here and use `'''` (since  `SUB_BOILERPLATE` and `SUB_BOILERPLATE_END` already use `'''`)